gvfs-helper.c

```diff
@@ -961,10 +1004,8 @@ static void select_odb(void)
 
 	if (gvfs_shared_cache_pathname && *gvfs_shared_cache_pathname)
 		odb_path = gvfs_shared_cache_pathname;
-	else {
-		prepare_alt_odb(the_repository);
+	else
```

Comment:
Why was the call to `prepare_alt_odb` removed?

Reply:
In the original version I was following the pattern where anyone who wanted to walk the odb-list, first called that to ensure it was populated (whether it was needed or not).

Yesterday, I noticed that there was no way for it to not be initialized by the time we reach this point in the code, so i removed it -- less visual noise.

I'll revisit.

Reply:
i added it back.